llvm/test/CodeGen/AArch64/aarch64-eh_return.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llc < %s -O0 -mtriple aarch64-none-linux-gnu | FileCheck %s
```

Comment:
I feel like this would be better as a MIR test that checks for `AArch64::ADDXrx64` specfically?